### PR TITLE
[documentation] #232: Update READMEs

### PR DESCRIPTION
**Pull request by outoftardis**
Signed-off-by: Ekaterina Mekhnetsova <mekkatya@gmail.com>

Partially addresses #232 

This PR only contains changes to Markdown files. There will be another PR for doc comments in code.

### CODE REVIEW

File: modules/block/README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-  import jp.co.soramitsu.iroha2.iroha2Mapper
-  ```
\ No newline at end of file
+The `block` module contains:
+
+- Genesis classes and some generated serializers and deserializers for data models
+- Serializers necessary to serialise Rust-specific models to Kotlin models
+- Deserializers necessary to deserialise models into JSON format, which is the format Iroha2 expects, for example, for [genesis.json](../testcontainers/src/main/resources/genesis.json)
+
+Note that you can also use `iroha2Mapper` from [deserializers.kt](./src/main/kotlin/jp/co/soramitsu/iroha2/deserializers.kt) <!-- The link is broken --> to deserialise models into JSON:
```
Comment (outoftardis) [inline modules/block/README.md:9]: The link is broken, and I couldn't find `iroha2Mapper` anywhere in this repo so I'm not sure where to link to

Comment (Mingela) [reply modules/block/README.md:9]: @gv-timur

File: README.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 
 - [x] [SDK](./modules/client) to work with Iroha2 API
 - [x] [codegen](./modules/codegen) to generate models into [model](./modules/model) by Iroha2 schema
-- [x] [model](./modules/model) generated models by [codegen](./modules/codegen)
-- [x] [block](./modules/block) genesis classes and some generated datamodels serializers and deserializers
-- [x] [testcontainers](./modules/testcontainers) wrapper for convenient integration testing with Iroha
+- [x] [model](./modules/model) with models generated by [codegen](./modules/codegen)
+- [x] [block](./modules/block) genesis classes and some generated serializers and deserializers for data models
+- [x] [testcontainers](./modules/testcontainers) wrapper for convenient integration testing with Iroha2
+
```
Comment (Mingela) [inline README.md:12]: image
some more to add I guess

**Review by Mingela — CHANGES_REQUESTED** [review]
Thanks for the contribution. This is just to resolve the comments first

File: README.md (hunk 2 above)
Comment (outoftardis) [reply README.md:14]: added

**Review by Mingela — APPROVED** [review]

**Comment by Mingela** [comment]
@outoftardis feel free to rerequest a review instead of dismissing the previous one (this is rather to use when a reviewer is unavailable for some reason)

**Review by arndey — APPROVED** [review]

**Pull request merged**